src/WebJobs.Script/SlidingWindow.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class SlidingWindow<TItem>
```

Comment:
Is the name here accurate? This class itself doesn't represent a sliding window.

Reply:
Yes, I think it is accurate. It is a time window of fixed width (ctor param) and the window "slides forward" in the sense that expired events are removed from the head and new events are added to the tail. Also in this sense: https://stackoverflow.com/questions/8269916/what-is-sliding-window-algorithm-examples